StabilityClient.Net/StabilityClient.Net.sln

```diff
@@ -12,5 +14,9 @@ Global
 		{4F5ABEB5-02DC-436F-A736-0BC23AEA2F33}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{4F5ABEB5-02DC-436F-A736-0BC23AEA2F33}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{4F5ABEB5-02DC-436F-A736-0BC23AEA2F33}.Release|Any CPU.Build.0 = Release|Any CPU
+		{23A67BFD-D323-430D-8FE1-14CC58B4D2EA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{23A67BFD-D323-430D-8FE1-14CC58B4D2EA}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{23A67BFD-D323-430D-8FE1-14CC58B4D2EA}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{23A67BFD-D323-430D-8FE1-14CC58B4D2EA}.Release|Any CPU.Build.0 = Release|Any CPU
 	EndGlobalSection
 EndGlobal
```

Comment:
Out of curiosity: in c# you need to add *.sln files to repo? In c++ the good practice is to use cmake that can generate solution/code for any platform if needed.